.circleci/workflows.yml

```diff
@@ -468,10 +468,9 @@ jobs:
 workflows:
   npm-release:
     when: 
-     condition: 
       or: 
-        - <<pipeline.parameters.deployment-artifacts-updated>>
-        - <<pipeline.parameters.deployment-package-json-updated>>
+        - equal: [ true, <<pipeline.parameters.deployment-artifacts-updated>> ]
+        - equal: [ true, <<pipeline.parameters.deployment-package-json-updated>> ]
```

Comment:
Nitpick: why is `equal` necessary here?
Isn't this the same as the expression it replaces? At least that is my impression from reading https://circleci.com/docs/configuration-reference/#logic-statements